### DISCUSSION TITLE
Key bindings of same ruleset should not allow conflicting

### DISCUSSION THREAD

**Discussion by harrumaki** [ppy/osu#13033] [Ideas]
Title. When streaming/doing faster alternating rhythms, only the "dominant" key (k1 for me) alternates while the "secondary" key is treated as if it were held (k2 for me). Harware is not the issue, works perfectly fine on stable.

**Comment by harrumaki** [comment]
[https://gyazo.com/b9d706d2357388c0e58acc4a05e01e6f](url)

**Comment by frenzibyte** [comment]
pressing Key 1 counts for **B1 and B2**? also what happens if you press the Key 2 without holding it?

**Comment by harrumaki** [comment]
yes, singletapping k1 is treated as a press for both b1 and b2 simultaneously. As for k2, it does the same. Streaming results in one of the keys being treated like they are held. This issue makes streaming impossible.

**Comment by frenzibyte** [comment]
can you show me your key map settings

**Comment by smoogipoo** [comment]
Have you accidentally bound the keys to each other in the settings?

**Comment by frenzibyte** [comment]
even so, we shouldn't allow for 2 buttons to be mapped to the same key

**Comment by harrumaki** [comment]
I will check when I get home.

**Comment by harrumaki** [comment]
K1 was bound to the default M2 slot. (https://gyazo.com/c3468e39d6edaca5e8ef67d96194d9d5)
My bad. However, as @frenzibyte has said, such a binding should be invalid. Thanks for the help.

**Comment by harrumaki** [comment]
I'll leave this opened in case this issue hasn't been flagged for fixing yet.
